src/js/helpers/constants.js

```diff
@@ -15,6 +15,16 @@ export const WEEKDAY_MAP = {
   SATURDAY: 6,
 };
 
-export const FULL_WIDTH_RESPONSIVE_IMAGE_SIZES = 
+export const DAYS = () => ([
```

Comment:
why a function and not an array?

Reply:
Excellent question! Maybe a strange design choice that we wouldn't want to adopt, but there is a reason for it.

When we import constants, they are mutable. If you require a constant and change it `DAYS.push(x), DAYS.splice(y)` that will actually change the original reference. So if another file or process requires that file, they'll receive the mutated version, not the original `DAYS`.

This was a tricky bug that I ran into in my days as a young developer.

So if we return a distinct function that returns an array, every instance of `import DAYS from ...; const days = DAYS();` will return a distinct object.

The alternative would be to remember to destruture DAYS on every invocation `{...DAYS}` to make a copy. But someone forgets to do that, then something will break.

It's a safeguard to make our constant work as expected. I'm open to other suggestions about it.

Reply:
makes perfect sense to me. thanks for explaining it so well